onebusaway-android/src/main/res/layout/arrivals_list_header_row_template.xml

```diff
@@ -2,145 +2,107 @@
+                android:layout_toRightOf="@id/eta_route_name"
+                android:layout_alignBaseline="@id/eta_route_name"
+                android:gravity="bottom"/>
+    </RelativeLayout>
 
-        <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                        android:id="@+id/eta_and_min"
-                        android:layout_width="wrap_content"
-                        android:layout_height="wrap_content"
-                        android:layout_column="2"
-                        android:layout_marginLeft="@dimen/arrival_header_text_pad_left"
-                        android:layout_marginTop="3dp"
-                        android:layout_marginBottom="3dp"
-                        android:paddingLeft="4dp"
-                        android:paddingRight="4dp"
-                        android:gravity="right"
-                        android:orientation="horizontal"
-                        android:background="@drawable/round_corners_style_b_header_status">
-                <TextView
-                        android:id="@+id/eta"
-                        android:layout_width="wrap_content"
-                        android:layout_height="wrap_content"
-                        android:text="2"
-                        android:textColor="@color/header_text_color"
-                        android:textStyle="bold"
-                        android:ellipsize="end"
-                        android:singleLine="true"/>
-                <TextView
-                        android:id="@+id/eta_min"
-                        android:layout_width="wrap_content"
-                        android:layout_height="wrap_content"
-                        android:textSize="14sp"
-                        android:text="min"
-                        android:textColor="@color/header_text_faded_color"
-                        android:ellipsize="end"
-                        android:singleLine="true"
-                        android:textIsSelectable="false"
-                        android:layout_marginLeft="5dp"
-                        android:layout_toRightOf="@id/eta"
-                        android:layout_alignBaseline="@id/eta"
-                        android:gravity="bottom"/>
-                <include
-                        android:id="@+id/eta_realtime_indicator"
-                        layout="@layout/realtime_indicator"
-                        android:layout_width="6dp"
-                        android:layout_height="6dp"
-                        android:layout_toRightOf="@id/eta_min"
-                        android:layout_marginTop="16dp"
-                        android:layout_marginLeft="1dp"/>
-        </RelativeLayout>
+    <include
+            android:id="@+id/eta_and_min"
+            layout="@layout/eta_header_view"
+            android:layout_marginTop="3dp"
+            android:layout_marginBottom="3dp"
+            android:layout_height="wrap_content"
+            android:layout_width="wrap_content"/>
```

Comment:
@cagryInside Could you put the `RelativeLayout` attributes here that are in the included view layout?

It should look something like:

~~~
    <include
            android:id="@+id/eta_and_min"
            layout="@layout/eta_header_view"
            android:layout_height="wrap_content"
            android:layout_width="wrap_content"
            android:layout_column="2"
            android:layout_marginLeft="@dimen/arrival_header_text_pad_left"
            android:layout_marginTop="3dp"
            android:layout_marginBottom="3dp"
            android:paddingLeft="4dp"
            android:paddingRight="4dp"
            android:gravity="right"
            android:orientation="horizontal"/>
~~~

(This is what I tested with - it seems `android:background="@drawable/round_corners_style_b_header_status"` is inherited from the included layout without a problem)

We need to list these attributes in this `<include>`, otherwise not all of them get carried over to the final layout (this isn't documented very well in Android, and IIRC inheritance behavior can be different for each type of attribute).  There are very small but visible differences in the padding and compression behavior of the table columns.  It's really hard to replicate this issue and compare against `master` branch because you need to catch the exact same combination of trip headsign and single/double digit ETAs to get the exact same layout result.  I wasn't convinced myself that the behavior was different until I caught the right headsign/ETAs :).

Here's an image created by layering a transparent screenshot above a solid screenshot with the same route/headsign, and very similar ETAs:
![image](https://cloud.githubusercontent.com/assets/928045/14755832/f2f9b5e2-08b2-11e6-813e-99832eec1ed6.png)

The transparent image on top is the one without the additional layout attributes included (i.e., this PR currently).  You can see that the colored box surrounding the ETA is slightly wider on the left than the ETA box in the solid image underneath, which includes all the attributes in the `<include>` element.

So, by including all attributes in the `<include>` element, it results in a slightly more compact ETA box.  I believe this behavior should match the `master` branch, although additional testing is welcome :).